main.go

```diff
@@ -300,3 +322,11 @@ func deleteTempFiles(directory string) error {
 	}
 	return nil
 }
+
+func diskSpaceCheck(mb uint64) (bool, error) {
+	var fs syscall.Statfs_t
+	if err := syscall.Statfs("/", &fs); err != nil {
```

Comment:
This shouldn't be assuming "/". The output directory from the configuration should be passed here in case that directory is a different volume from the root volume.